### PR TITLE
Execute tests on Node 12

### PR DESCRIPTION
**Pull request by kibertoad**

### CODE REVIEW

**Comment by kibertoad** [comment]
Currently blocked by sqlite support: https://github.com/mapbox/node-sqlite3/issues/1151

**Comment by DRoet** [comment]
FYI the upstream issue has been fixed in sqlite `4.0.7`

**Comment by kibertoad** [comment]
@elhigu Do you prefer to wait with merging this one until Docker tests one lands since this one would be easier to resolve conflicts for?

File: .travis.yml [hunk 1]
```diff
@@ -60,13 +60,13 @@ services:
 addons:
   postgresql: '9.6'
   apt:
-    sources:
-      - mysql-5.7-trusty
```
Comment (kibertoad) [inline .travis.yml]: MySQL 5.7 is default on Xenial, no explicit sources are needed.

File: .travis.yml [hunk 2]
```diff
@@ -60,13 +60,13 @@ services:
...
     packages:
       - g++-4.8
       - gcc-4.8
+      - g++-5
```
Comment (kibertoad) [inline .travis.yml]: new SQLite version requires gcc 5

**Comment by kibertoad** [comment]
End changes are not that intrusive (since xenial actually has needed gcc deps by default), so conflict resolution should be in either case quite trivial.

**Pull request merged**